static/app/views/settings/organizationMembers/components/inviteMembersButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import FeatureDisabled from 'sentry/components/acl/featureDisabled';
+import Button from 'sentry/components/button';
+import {Hovercard} from 'sentry/components/hovercard';
+import {IconMail} from 'sentry/icons';
+import {t} from 'sentry/locale';
+
+type InviteMembersButtonProps = {
+  onClick: () => void;
+  disabled?: boolean;
+};
+
+function InviteMembersButton({disabled, onClick}: InviteMembersButtonProps) {
+  const action = (
+    <Button
+      priority="primary"
+      size="small"
+      onClick={onClick}
+      data-test-id="email-invite"
+      icon={<IconMail />}
+      disabled={disabled}
+    >
+      {t('Invite Members')}
+    </Button>
+  );
+
+  return disabled ? (
+    <Hovercard
```

Comment:
If possible, can we actually keep this all co-located near the feature hook stuff?

I would probably write things similar to this https://github.com/getsentry/sentry/blob/3a09b483dc327a280d74ca7634f125f5a71fb659/static/app/views/dashboardsV2/controls.tsx#L172-L203

Reply:
sounds good. I inlined the `InviteMembersButton` component into organizationMembersWrapper.tsx